### PR TITLE
Update French translation

### PR DESCRIPTION
**Pull request by trebmuh**
Hope that helps.

### CODE REVIEW

File: res/main/i18n/fr/statuses.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
 	"jack": {
-		"label": "JACK:",
+		"label": "JACK :",
```
Comment (sadko4u) [inline res/main/i18n/fr/statuses.json:3]: This space looks suspicious. Isn't it an occasional key hit?

File: res/main/i18n/fr/messages.json [hunk 2]
```diff
@@ -1,22 +1,22 @@
...
 			"pkg_name": "{@meta_pkg_artifact_name}",
 			"pkg_version": "{@meta_pkg_version}",
 			"plug_name": "{@meta_plugin_description}",
 			"plug_version": "{@meta_plugin_version}"
 		},
-		"license": "Licensed under the terms of GNU Lesser Public License v3 (LGPLv3):",
+		"license": "Licencié selon les termes de la GNU Lesser Public License v3 (LGPLv3) :",
 		"license_url": "https://www.gnu.org/licenses/lgpl-3.0.fr.html",
-		"support": "Support the project:"
+		"support": "Supporter le projec :"
```
Comment (sadko4u) [inline res/main/i18n/fr/messages.json:19]: Also see an extra space here

File: res/main/i18n/fr/statuses.json (hunk 1 above)
Comment (trebmuh) [reply res/main/i18n/fr/statuses.json:3]: Nope, it's correct.
It's a French language rule: Spaces before (most) punctuations.

File: res/main/i18n/fr/messages.json (hunk 2 above)
Comment (trebmuh) [reply res/main/i18n/fr/messages.json:19]: the space is fine, but I see I made a typo mistake.
I'll correct that in a few seconds.

**Comment by sadko4u** [comment]
Merged changes into `devel` branch: 1e7556c6c3c2cbd8f09f32b9b7ba0773508fafc0

How about syncing french translations for plugins?
Now plugins also have their own language bundles in their repositories.

**Comment by trebmuh** [comment]
Why not, I'll do so when I have time.

**Pull request closed without merging**